py/ngff_zarr/v06/__init__.py

```
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: Copyright (c) Fideus Labs LLC
+# SPDX-License-Identifier: MIT
```

Ruff (0.15.12)

[warning] No newline at end of file

Add trailing newline

(W292)